### PR TITLE
[hi] Localize application-programming-interface.md to Hindi

### LINKED ISSUE #1624: Localize Application Programming Interface (API) page in Hindi
The application Programming Interface (API) page is available in English and Bengali. Still, This term also needs to be Localized into Hindi for the people who want to read this term in Hindi.
Link: https://glossary.cncf.io/application-programming-interface/

### PR DESCRIPTION
**Pull request by SaxenaAnushka102**
Signed-off-by: ANUSHKA SAXENA <anushkasaxena102@gmail.com>

### Describe your changes

Created a file called application-programming-interface.md to localize the page: https://glossary.cncf.io/application-programming-interface/ into Hindi 

### Related issue number or link 
Resolves #1624

### Checklist before opening this PR 
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/hi/application-programming-interface.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: अनुप्रयोग प्रोग्रामिंग का अंतरापृष्ठ (Application Programming Interface)
+status: Feedback Appreciated
+category: संकल्पना
+tags:  ["संरचना","मूलभूत"]
+---
```
Comment (anubha-v-ardhan) [inline content/hi/application-programming-interface.md:6]:
```suggestion
---
title: एप्लीकेशन प्रोग्रामिंग इंटरफ़ेस (Application Programming Interface)
status: Completed
category: संकल्पना
tags: ["architecture", "fundamental", ""]
---
```

File: content/hi/application-programming-interface.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+## यह क्या है
+
+एक एपीआई कंप्यूटर प्रोग्राम के लिए संवाद स्थापित करने का एक माध्यम है। जिस तरह मनुष्य एक वेब पेज के माध्यम से एक वेबसाइट के साथ सूचना का आदान-प्रदान करते हैं, एक एपीआई विभिन्न कंप्यूटर प्रोग्राम को एक दूसरे को सूचना का आदान-प्रदान करने की अनुमति देता है।  मानवीय अंतःक्रियाओं के विपरीत, एपीआई की सीमाएँ होती हैं कि उनसे क्या पूछा जा सकता है और क्या नहीं। सहभागिता के बीच सीमा स्थापित करने से अलग-अलग प्रोग्राम के बीच स्थिर और कार्यात्मक संचार बनाने में मदद मिलती है।
+
+## समस्या जिस का यह समाधान करता है
```
Comment (anubha-v-ardhan) [inline content/hi/application-programming-interface.md:12]:
```suggestion
## समस्या
```

File: content/hi/application-programming-interface.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+जैसे-जैसे एप्लिकेशन अधिक जटिल होते जाते हैं, कोड में छोटे परिवर्तन भी दूसरी कार्यक्षमताओं पर भारी प्रभाव डाल सकते हैं। अनुप्रयोगों को उनकी कार्यक्षमता के लिए एक प्रतिरुपक दृष्टिकोण रखने की आवश्यकता होती है, यदि वे एक साथ विकास और स्थिरता बनाए रख सकते हैं।  एपीआई के बिना, अनुप्रयोगों के बीच बातचीत के लिए एक रूपरेखा का अभाव रहता है। एक साझा ढांचे के बिना, अनुप्रयोगों को [स्केल](/scalability/) और एकीकृत करना चुनौतीपूर्ण है।
+
+## यह कैसे मदद करता है
```
Comment (anubha-v-ardhan) [inline content/hi/application-programming-interface.md:16]:
```suggestion
## समाधान
```

File: content/hi/application-programming-interface.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+---
+title: अनुप्रयोग प्रोग्रामिंग का अंतरापृष्ठ (Application Programming Interface)
+status: Feedback Appreciated
+category: संकल्पना
+tags:  ["संरचना","मूलभूत"]
+---
+
+## यह क्या है
+
+एक एपीआई कंप्यूटर प्रोग्राम के लिए संवाद स्थापित करने का एक माध्यम है। जिस तरह मनुष्य एक वेब पेज के माध्यम से एक वेबसाइट के साथ सूचना का आदान-प्रदान करते हैं, एक एपीआई विभिन्न कंप्यूटर प्रोग्राम को एक दूसरे को सूचना का आदान-प्रदान करने की अनुमति देता है।  मानवीय अंतःक्रियाओं के विपरीत, एपीआई की सीमाएँ होती हैं कि उनसे क्या पूछा जा सकता है और क्या नहीं। सहभागिता के बीच सीमा स्थापित करने से अलग-अलग प्रोग्राम के बीच स्थिर और कार्यात्मक संचार बनाने में मदद मिलती है।
```
Comment (anubha-v-ardhan) [inline content/hi/application-programming-interface.md:10]:
```suggestion
एक एपीआई कंप्यूटर प्रोग्राम के लिए संवाद स्थापित करने का एक माध्यम है। जिस तरह मनुष्य एक वेब पेज के माध्यम से एक वेबसाइट के साथ सूचना का आदान-प्रदान करते हैं, एक एपीआई विभिन्न कंप्यूटर प्रोग्राम को एक दूसरे को सूचना का आदान-प्रदान करने की अनुमति देता है। मानवीय अंतःक्रियाओं के विपरीत, एपीआई की सीमाएँ होती हैं कि उनसे क्या पूछा जा सकता है और क्या नहीं। सहभागिता के बीच सीमा स्थापित करने से अलग-अलग प्रोग्राम के बीच स्थिर और कार्यात्मक संचार बनाने में मदद मिलती है।
```

File: content/hi/application-programming-interface.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+## समस्या जिस का यह समाधान करता है
+
+जैसे-जैसे एप्लिकेशन अधिक जटिल होते जाते हैं, कोड में छोटे परिवर्तन भी दूसरी कार्यक्षमताओं पर भारी प्रभाव डाल सकते हैं। अनुप्रयोगों को उनकी कार्यक्षमता के लिए एक प्रतिरुपक दृष्टिकोण रखने की आवश्यकता होती है, यदि वे एक साथ विकास और स्थिरता बनाए रख सकते हैं।  एपीआई के बिना, अनुप्रयोगों के बीच बातचीत के लिए एक रूपरेखा का अभाव रहता है। एक साझा ढांचे के बिना, अनुप्रयोगों को [स्केल](/scalability/) और एकीकृत करना चुनौतीपूर्ण है।
```
Comment (anubha-v-ardhan) [inline content/hi/application-programming-interface.md:14]:
```suggestion
जैसे-जैसे एप्लिकेशन अधिक जटिल होते जाते हैं, कोड में छोटे परिवर्तन भी दूसरी कार्यक्षमताओं पर भारी प्रभाव डाल सकते हैं। एप्लीकेशन को उनकी कार्यक्षमता के लिए एक प्रतिरुपक दृष्टिकोण रखने की आवश्यकता होती है, यदि वे एक साथ विकास और स्थिरता बनाए रख सकते हैं। एपीआई के बिना, एप्लीकेशन के बीच बातचीत के लिए एक रूपरेखा का अभाव रहता है। एक साझा ढांचे के बिना, एप्लीकेशन को [स्केल](/scalability/) और एकीकृत करना चुनौतीपूर्ण है।
```

File: content/hi/application-programming-interface.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+
+## यह कैसे मदद करता है
+
+एपीआई कंप्यूटर प्रोग्राम या एप्लिकेशन को एक निर्धारित और समझने योग्य तरीके से जानकारी साझा करने और साझा करने की अनुमति देता है। ये आधुनिक अनुप्रयोगों के लिए एक मूलभूत अंग हैं और ये डेवलपर्स को अनुप्रयोगों को एक साथ एकीकृत करने का एक तरीका प्रदान करते हैं। जब भी आप एक साथ काम करने वाली [माइक्रोसर्विसेज]((/microservices/)) के बारे में सुनते हैं, तो आप अनुमान लगा सकते हैं कि वे एक एपीआई के माध्यम से संवाद करती हैं।
```
Comment (anubha-v-ardhan) [inline content/hi/application-programming-interface.md:18]:
```suggestion
एपीआई कंप्यूटर प्रोग्राम या एप्लिकेशन को एक निर्धारित और समझने योग्य तरीके से जानकारी साझा करने और साझा करने की अनुमति देता है। ये आधुनिक एप्लीकेशन के लिए एक मूलभूत अंग हैं और ये डेवलपर्स को एप्लीकेशन को एक साथ एकीकृत करने का एक तरीका प्रदान करते हैं। जब भी आप एक साथ काम करने वाली [माइक्रोसर्विसेज]((/microservices/)) के बारे में सुनते हैं, तो आप अनुमान लगा सकते हैं कि वे एक एपीआई के माध्यम से संवाद करती हैं।
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]

File: content/hi/application-programming-interface.md (hunk 6 above)
Comment (SaxenaAnushka102) [reply content/hi/application-programming-interface.md]: Hi @anubha-v-ardhan! I've incorporated these changes in the latest commits. Please let me know if there's anything that can be improved. Thanks.

**Comment by anubha-v-ardhan** [comment]
Hey @SaxenaAnushka102, DCO check is failing on this PR which most probably mean commits are not properly signed. You can view more [detail here](https://github.com/cncf/glossary/pull/1632/checks?check_run_id=14696620660). Kindly fix the issue if possible, so we can proceed with approval.

**Comment by SaxenaAnushka102** [comment]
Hi @anubha-v-ardhan! I did git reset to the initial commit & then force pushed the commit with correct mail ID. Still I'm getting this error:
Commit sha: [402c004](https://github.com/cncf/glossary/pull/1632/commits/402c0043d65b95ecd938fc944a634a109a9e70bc), Author: ANUSHKA SAXENA, Committer: ANUSHKA SAXENA; The sign-off is missing.

**Comment by SaxenaAnushka102** [comment]
Hi @anubha-v-ardhan! I've tried my best to resolve the failed check & update the requested changes. Please let me know if any further changes are needed here.

File: content/hi/application-programming-interface.md [hunk 7]
```diff
@@ -0,0 +1,18 @@
+---
+title: एप्लीकेशन प्रोग्रामिंग इंटरफ़ेस (Application Programming Interface)
+status: Completed
+category: संकल्पना
+tags: ["architecture", "fundamental"]
+---
+
+## यह क्या है
+
+एक एपीआई कंप्यूटर प्रोग्राम के लिए संवाद स्थापित करने का एक माध्यम है। जिस तरह मनुष्य एक वेब पेज के माध्यम से एक वेबसाइट के साथ सूचना का आदान-प्रदान करते हैं, एक एपीआई विभिन्न कंप्यूटर प्रोग्राम को एक दूसरे को सूचना का आदान-प्रदान करने की अनुमति देता है। मानवीय अंतःक्रियाओं के विपरीत, एपीआई की सीमाएँ होती हैं कि उनसे क्या पूछा जा सकता है और क्या नहीं। सहभागिता के बीच सीमा स्थापित करने से अलग-अलग प्रोग्राम के बीच स्थिर और कार्यात्मक संचार बनाने में मदद मिलती है।
```
Comment (aj11anuj) [inline content/hi/application-programming-interface.md:10]:
```suggestion
यह एक एपीआई कंप्यूटर प्रोग्राम के लिए संवाद स्थापित करने का एक माध्यम है। जिस तरह हम एक वेब पेज के माध्यम से एक वेबसाइट के साथ सूचना का आदान-प्रदान करते हैं, एक एपीआई विभिन्न कंप्यूटर प्रोग्राम को एक दूसरे से सूचना का आदान-प्रदान करने के लिए उपयोग करती है। अंतः मानवीय क्रियाओं के विपरीत, एपीआई की सीमाएँ होती हैं कि उनसे क्या पूछा जा सकता है और क्या नहीं। सहभागिता के बीच सीमा स्थापित करने से अलग-अलग प्रोग्राम के बीच स्थिर और कार्यात्मक संचार बनाने में मदद मिलती है।
```

File: content/hi/application-programming-interface.md [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+
+## समस्या
+
+जैसे-जैसे एप्लिकेशन अधिक जटिल होते जाते हैं, कोड में छोटे परिवर्तन भी दूसरी कार्यक्षमताओं पर भारी प्रभाव डाल सकते हैं। एप्लीकेशन को उनकी कार्यक्षमता के लिए एक प्रतिरुपक दृष्टिकोण रखने की आवश्यकता होती है, यदि वे एक साथ विकास और स्थिरता बनाए रख सकते हैं। एपीआई के बिना, एप्लीकेशन के बीच बातचीत के लिए एक रूपरेखा का अभाव रहता है। एक साझा ढांचे के बिना, एप्लीकेशन को [स्केल](/scalability/) और एकीकृत करना चुनौतीपूर्ण है।
+
+## समाधान
+
+एपीआई कंप्यूटर प्रोग्राम या एप्लिकेशन को एक निर्धारित और समझने योग्य तरीके से जानकारी साझा करने और साझा करने की अनुमति देता है। ये आधुनिक एप्लीकेशन का एक मूलभूत अंग हैं और ये डेवलपर्स को विभिन्न एप्लीकेशन को एक साथ एकीकृत करने का एक तरीका प्रदान करते हैं। जब भी आप एक साथ काम करने वाली [माइक्रोसर्विसेज]((/microservices/)) के बारे में सुनते हैं, तो आप अनुमान लगा सकते हैं कि वे एक एपीआई के माध्यम से संवाद करती हैं।
```
Comment (aj11anuj) [inline content/hi/application-programming-interface.md:18]:
```suggestion
एपीआई कंप्यूटर प्रोग्राम या एप्लिकेशन को एक निर्धारित और समझने योग्य तरीके से जानकारी साझा करने की अनुमति देता है। ये आधुनिक एप्लीकेशन का एक मूलभूत अंग हैं और ये डेवलपर्स को विभिन्न एप्लीकेशन को एक साथ एकीकृत करने का एक तरीका प्रदान करता है। जब भी आप एक साथ काम करने वाली [माइक्रोसर्विसेज](/microservices/) के बारे में सुनते हैं, तो आप अनुमान लगा सकते हैं कि वे एक एपीआई के माध्यम से संवाद करती हैं।
```

File: content/hi/application-programming-interface.md [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+tags: ["architecture", "fundamental"]
+---
+
+## यह क्या है
+
+एक एपीआई कंप्यूटर प्रोग्राम के लिए संवाद स्थापित करने का एक माध्यम है। जिस तरह मनुष्य एक वेब पेज के माध्यम से एक वेबसाइट के साथ सूचना का आदान-प्रदान करते हैं, एक एपीआई विभिन्न कंप्यूटर प्रोग्राम को एक दूसरे को सूचना का आदान-प्रदान करने की अनुमति देता है। मानवीय अंतःक्रियाओं के विपरीत, एपीआई की सीमाएँ होती हैं कि उनसे क्या पूछा जा सकता है और क्या नहीं। सहभागिता के बीच सीमा स्थापित करने से अलग-अलग प्रोग्राम के बीच स्थिर और कार्यात्मक संचार बनाने में मदद मिलती है।
+
+## समस्या
+
+जैसे-जैसे एप्लिकेशन अधिक जटिल होते जाते हैं, कोड में छोटे परिवर्तन भी दूसरी कार्यक्षमताओं पर भारी प्रभाव डाल सकते हैं। एप्लीकेशन को उनकी कार्यक्षमता के लिए एक प्रतिरुपक दृष्टिकोण रखने की आवश्यकता होती है, यदि वे एक साथ विकास और स्थिरता बनाए रख सकते हैं। एपीआई के बिना, एप्लीकेशन के बीच बातचीत के लिए एक रूपरेखा का अभाव रहता है। एक साझा ढांचे के बिना, एप्लीकेशन को [स्केल](/scalability/) और एकीकृत करना चुनौतीपूर्ण है।
```
Comment (aj11anuj) [inline content/hi/application-programming-interface.md:14]:
```suggestion
जैसे-जैसे एप्लिकेशन अधिक जटिल होते जाते हैं, कोड में छोटे परिवर्तन भी दूसरी कार्यक्षमताओं पर भारी प्रभाव डालने लगते हैं। एप्लीकेशन को उनकी कार्यक्षमता के लिए एक प्रतिरुपक दृष्टिकोण रखने की आवश्यकता होती है, यदि वे एक साथ विकास और स्थिरता बनाए रख सकते हैं। एपीआई के बिना, एप्लीकेशन के बीच बातचीत के लिए एक रूपरेखा का अभाव रहता है। एक साझा ढांचे के बिना, एप्लीकेशन को [स्केल](/scalability/) और एकीकृत करना चुनौतीपूर्ण है।
```

**Review by aj11anuj — CHANGES_REQUESTED** [review]
Some suggestions from my side

**Review by aj11anuj — APPROVED** [review]
Lgtm 👍

**Pull request merged**